### ISSUE TITLE
Sign up to parking system

### ISSUE THREAD

**Issue by ofiralexi** [TechnionYP5779/team1#189]
Description: Create a suscribe platform to the parking project. Make sure you insert the data to the DB, and document the fields.

**Comment by yoavz1997** [comment]
The initial design was pushed and merged into development. After the other UI components are finished, a backend support will be added in a different branch.

**Comment by yoavz1997** [comment]
Closed after #208 and #214 were merged
